### DISCUSSION TITLE
[Resolved]The number of my notes always shows as 0

### DISCUSSION THREAD

**Discussion by lillian-who** [st3v3nmw/obsidian-spaced-repetition#378] [Q&A]
![图片](https://user-images.githubusercontent.com/20553685/150263024-812884d0-124f-4d39-bf03-a227e214996a.png)

I set the note review. Why does 0 always appear here? 

How can I get it to show how many notes I should review today?

**Comment by Bradford-Eaton** [comment]
@lillian-who Projects like this thrive on collaboration and transparent feedback. Others likely have the same issue. Kindly, *if your answer is resolved please comment on the solution offered or found*
